scripts/prepare_dutchf3.py

```diff
@@ -222,29 +277,33 @@ def split_alaudah_et_al_19(data_dir, stride, fraction_validation=0.2, loader_typ
         x_list = ["x_" + str(x) for x in range(xline)]
     elif loader_type == "patch":
         i_list = []
-        horz_locations = range(0, xline - stride, stride)
-        vert_locations = range(0, depth - stride, stride)
+        horz_locations = range(0, xline - patch_size + 1, stride)
+        vert_locations = range(0, depth - patch_size + 1, stride)
```

Comment:
How does this change is related to  skip_steps?  Should it be part of a separate PR?

Reply:
Are we going to open an issue & PR just to drop this method? If it should be dropped, I could open an issue and link that to this PR. Do you agree?